packages/account-v2/src/containers/POAForm/DocumentSubmission/DocumentSubmission.tsx

```diff
@@ -16,33 +15,33 @@ const listItems = [
 const documentValidation = Yup.mixed().required('Please upload a file');
 
 const DocumentSubmission: React.FC = () => {
-    const { isMobile } = useBreakpoint();
+    const { isMobile } = useDevice();
```

Comment:
```suggestion
    const { isDesktop } = useDevice();
```

Reply:
`isDesktop` and `isMobile` will be handle in the next PR later 🙏